Dolphin scripts/Entrance Randomizer/CONFIGS.py

```diff
@@ -6,7 +6,7 @@
 """
 Set your own seed, can be an `int`, `float`, `str`, `bytes` or `bytearray`.
 
-Use `None` or any Falsy value to generate a random seed.
+Use `None` or any False value to generate a random seed.
```

Comment:
`Truthy` means would evaluate to `True` in a condition
```suggestion
Use `None` or any Falsy value to generate a random seed.
```